File: app/src/main/java/tech/diarmaid/koohiiaite/interfaces/OnCSVParseCompleted.java
package tech.diarmaid.koohiiaite.interfaces;

import tech.diarmaid.koohiiaite.model.CSVEntry;

import java.util.List;

/**
 * Callback to update table containing csv contents after
 * parsing completed
 */
public interface OnCSVParseCompleted {
    void onParsingCompleted(List<CSVEntry> parsedEntries);
}

File: app/src/main/java/tech/diarmaid/koohiiaite/interfaces/OnCSVParseCompleted.kt
package tech.diarmaid.koohiiaite.interfaces

import tech.diarmaid.koohiiaite.model.CSVEntry

/**
 * Callback to update table containing csv contents after
 * parsing completed
 */
interface OnCSVParseCompleted {
    fun onParsingCompleted(parsedEntries: List<CSVEntry>)
}
